ipyleaflet/leaflet.py

```diff
@@ -549,16 +549,18 @@ class TileLayer(RasterLayer):
     url: string, default "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
         Url to the tiles service.
     min_zoom: int, default 0
-        Minimum zoom for this tile service.
+        The minimum zoom level down to which this layer will be displayed (inclusive).
     max_zoom: int, default 18
-        Maximum zoom for this tile service.
-    min_native_zoom: int, default 0
-    max_native_zoom: int, default 18
+        The maximum zoom level up to which this layer will be displayed (inclusive). Note that if you want to display a high resolution tile layer with a large zoom, you have to set max_zoom and max_native_zoom with equal value (An example is given in example/Max_zoom_tests.ipynb).
```

Comment:
```suggestion
        The maximum zoom level up to which this layer will be displayed (inclusive).
```

The rest of the comment was not technically true. One can set max_zoom and max_native_zoom with different values and still be able to zoom in on a high resolution tile layer.

Also, pointing to external examples may lead to broken links when renaming/moving example notebooks.